images/win/scripts/Installers/Validate-Toolset.ps1

```diff
@@ -33,7 +37,11 @@ function Validate-SystemDefaultTool {
 
     # Check if tool on path
     if (Get-Command -Name $binName) {
-        $versionOnPath = $(& $binName --version 2>&1) | Select-String -Pattern ".*(\d+\.\d+\.\d+)"
+        $versionCommand = "--version"
+        If ($binName -Match "go") {
+            $versionCommand = "version"
+        }

```

Comment:
Can we put this information to 
```
$toolsExecutables = @{
    Python = @("python.exe", "Scripts\pip.exe")
    node = @("node.exe", "npm")
    PyPy = @("python.exe", "Scripts\pip.exe")
    go = @("bin\go.exe")
}
```
somehow?

Reply:
Done.